product_docs/docs/postgres_distributed_for_kubernetes/1/before_you_start.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: 'Before you start'
+originalFilePath: 'src/before_you_start.md'
+---
+
+Before you get started, it's essential that you become familiar with some terminology that's
+specific to Kubernetes and PGD.
+
+## Kubernetes terminology
+
+[Node](https://kubernetes.io/docs/concepts/architecture/nodes/)
+: A *node* is a worker machine in Kubernetes, either virtual or physical, where
+  all services necessary to run pods are managed by the control plane nodes.
+
+[Pod](https://kubernetes.io/docs/concepts/workloads/pods/pod/)
+: A *pod* is the smallest computing unit that can be deployed in a Kubernetes
+  cluster and is composed of one or more containers that share network and
+  storage.
+
+[Service](https://kubernetes.io/docs/concepts/services-networking/service/)
+: A *service* is an abstraction that exposes as a network service an
+  application that runs on a group of pods and standardizes important features,
+  such as service discovery across applications, load balancing, and failover.
+
+[Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
+: A *secret* is an object that's designed to store small amounts of sensitive
+  data, such as passwords, access keys, or tokens, and use them in pods.
```

Comment:
```suggestion
  data, such as passwords, access keys, or tokens, and uses them in pods.
```
I am assuming the subject here is the secret?

Reply:
Neither of these is quite accurate; a secret doesn't itself use anything, rather it is available for use. And while secrets could be used for data exchange, there's no particular requirement that any pod be able to update them. I'm gonna suggest 

```suggestion
  data, such as passwords, access keys, or tokens, for use within pods.
```

...as I think that's a definition that's accurate enough for the purposes of configuring PG4K-PGD (which is our goal here) - we link to the full docs for folks who want a more rigorous definition!